src/test/suites/login/loginTests.robot

```diff
@@ -0,0 +1,50 @@
+*** Settings ***
+Library  SeleniumLibrary
+Library  Collections
+Library  OperatingSystem
+Library  String
+
+Resource  ${CURDIR}/../../../../resources/conf/runners/keywords.robot
+Resource  ${CURDIR}/../../../app/pages/Login.robot
+Resource  ${CURDIR}/../../../app/pages/Checkout.robot
+Resource  ${CURDIR}/../../../app/pages/HomePage.robot
+Resource  ${CURDIR}/../../../app/pages/Confirmation.robot
+Resource  ${CURDIR}/../../../app/pages/Favourites.robot
+
+
+*** Test Cases ***
+#Test Case 3
+Check if Signin opens on clicking on favourites nav item 
+
+    Start Test  ${testType}
+
+    Run Keyword If    '${testType}'=='bstack-single'    Rename Session  Check Fav Nav    ELSE IF    '${testType}'=='bstack-local'    Rename Session  Check Fav Nav
+
+    Click On Favourites
+
+    Sleep   1
```

Comment:
better way that to use sleep - please use explicit waits

Reply:
removed, was added earlier and forgot to remove